app/globals/globals.go

```diff
@@ -0,0 +1,13 @@
+package globals
+
+import "errors"
+
+var (
+	ErrDuplicateEmail = errors.New("email is already taken")
+	ErrNotFound       = errors.New("no user found in DB")
+)
+
+const (
+	UniqueViolationErrCode    = "unique_violation"
```

Comment:
It looks like these code messages are specific to postgres. Is there an absolute necessity to separate them from the repository?

Reply:
remove